tests/Makefile.am

```diff
@@ -41,12 +41,13 @@ tmp.softhsm:
 
 dist_check_SCRIPTS = \
 	helpers.sh setup-softhsm.sh setup-softokn.sh softhsm-proxy.sh \
-	test-wrapper tbasic tcerts teccsha2 thkdf toaepsha2 trsapss
+	test-wrapper tbasic tcerts tecc tecdh thkdf toaepsha2 trsapss
 
 test_LIST = \
 	basic-softokn basic-softhsm-proxy \
 	certs-softokn certs-softhsm-proxy \
-	eccsha2-softokn \
+	ecc-softokn ecc-softhsm-proxy \
+	ecdh-softokn \
```

Comment:
the derive does not work with the softhsm (again?)

Reply:
nope, that is why I split it off

Reply:
would be worth investigating if it is something in the softhsm, openssl or here to fix. Can you open a new issue for that?